pkg/retry/retry.go

```diff
@@ -16,6 +16,6 @@ package retry
 import "time"
 
 const (
-	DefaultLinearBackoffInterval = time.Second
+	DefaultLinearBackoffInterval = time.Minute
```

Comment:
Same retry is used for remote actor invocation also here - https://github.com/dapr/dapr/blob/cc4df6b440b41f3b51e8253cc08dffb286ac1082/pkg/actors/actors.go#L332

Please let me know if retry interval for both needs to be different.